### DISCUSSION TITLE
Styles at posts and pages

### DISCUSSION THREAD

**Discussion by iamgiannisx** [razonyang/hugo-theme-bootstrap#568] [Q&A] [answered]
Is it possible to add custom styles in some line at markdown inside posts or pages ?

**Comment by razonyang** [comment]
Could you please explain it in detail?

**Comment by razonyang** [comment]
You can put your CSS into `assets/main/scss/_custom.scss`.  And then you can use shortcode or the HTML markup directly for generating special *line*.

**Comment by iamgiannisx** [comment]
I am trying to center text for some links [mylink](https://mylink.com) 
I create a shortcode 
center-text.html
`<p style="text-align: center;">{{ .Inner }}</p>` 
This shortcode works fine with plain text but is not working with links, I try also the way you mention earlier but I endup with the same result.

**Reply by razonyang** [reply]
I guess it's an issue about the `p` tag, you can check the generated  source code, does the generated links wrapped by `p` block?
If no, replace it with `div` would be OK.

**Comment by razonyang** [comment, marked as answer]
Btw, I recommend using Bootstrap utilities CSS instead of inline style, in this case, you can define something like following:

```html
<div class="text-center">
  {{ .Inner }}
</div>
```

> You may need to restart Hugo server.

If you prefer using Markdown inside a shortcode, you'll need to process the Markdown:

```html
{{< shortcodename >}}
[My Site](https://example.com)
{{</ shortcodename >}}
```

```html
{{ .Inner | markdownify }}
```

**Comment by iamgiannisx** [comment]
Thanks for the info, I totally forgot `markdownify`
